eng/helix/content/RunTests/Program.cs

```diff
@@ -19,14 +19,18 @@ static async Task Main(string[] args)
                 {
                     keepGoing = await runner.InstallDotnetToolsAsync();
                 }
-#if INSTALLPLAYWRIGHT
+
```

Comment:
I believe the reason I did this was because some queues don't support playwright but maybe now that we are building this on CI rather than on the actual machines, this doesn't matter anymore, but just something to be aware of